toolset/benchmark/framework_test.py

```diff
@@ -289,12 +292,22 @@ def verify_type(test_type):
       test.passed = all(result is 'pass' for (result, reason, url) in results)
       
       def output_result(result, reason, url):
-        out.write("   %s for %s\n" % (result.upper(), url))
-        print "   %s for %s" % (result.upper(), url)
+        specific_rules_url = "http://frameworkbenchmarks.readthedocs.org/en/latest/Project-Information/Framework-Tests/#specific-test-requirements"
+        color = Fore.GREEN
+        if result.upper() == "WARN":
+          color = Fore.YELLOW
+        elif result.upper() == "FAIL":
+          color = Fore.RED
+
+        out.write(("   " + color + "%s" + Style.RESET_ALL + " for %s\n") % (result.upper(), url))
+        print ("   " + color + "%s" + Style.RESET_ALL + " for %s") % (result.upper(), url)
         if reason is not None and len(reason) != 0:
           for line in reason.splitlines():
             out.write("     " + line + '\n')
             print "     " + line
+          if not test.passed:
+            out.write("     See %s" % specific_rules_url)
+            print "     See %s" % specific_rules_url
```

Comment:
Can we add a line break, so that [this doesn't happen](https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/57712944#L6263) and the top header line "-------------------------" gets bumped to a new line?

```bash
INFO:run-ci:   WARN for http://127.0.0.1:8080/entityframework/postgresql?queries=foo
INFO:run-ci:     Top-level JSON is an object, not an array
INFO:run-ci:     See http://frameworkbenchmarks.readthedocs.org/en/latest/Project-Information/Framework-Tests/#specific-test-requirements--------------------------------------------------------------------------------
INFO:run-ci:  VERIFYING UPDATE
INFO:run-ci:--------------------------------------------------------------------------------
```